src/meson.build

```
@@ -44,3 +44,10 @@ pkg_mod.generate(libraries : lc3lib,
                  name : 'liblc3',
                  filebase : 'lc3',
                  description : 'LC3 codec library')
+
+#Declare dependency
+liblc3_dep = declare_dependency(
+	link_with:lc3lib,
+	include_directories:inc)
+
+meson.override_dependency('liblc3', liblc3_dep)
```

Instead of bumping the minimum version (which is not available in the distro repositories for e.g. Debian oldstable or Ubuntu 20.04), you can do:

```suggestion
if meson.version().version_compare('>=0.54.0')
    meson.override_dependency('liblc3', liblc3_dep)
endif
```

---

Done